src/components/openHouse/page.tsx

```
@@ -0,0 +1,415 @@
+
+  const hasDemos = Object.values(orgsData).some((org) => org.demo_time != null);
+
+  return (
+    <div className="relative w-full px-4 pt-24 md:px-8 lg:pt-32">
+      {/* Calendar Modal */}
+      {isCalendarModalOpen && (
+        <div
+          className="fixed inset-0 z-50 flex animate-[fadeIn_200ms_ease-out] items-center justify-center bg-black/40 backdrop-blur-sm"
+          onClick={() => setIsCalendarModalOpen(false)}
+        >
+          <div
```

Biome (2.3.13)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)